lib/Importer/Caffe2.cpp

```diff
@@ -49,6 +49,28 @@ static ArgumentDictionaryTy loadArgumentMap(const caffe2::OperatorDef &op) {
   return dict;
 }
 
+static std::vector<size_t> getPads(ArgumentDictionaryTy &dict) {
+  std::vector<size_t> zeroPads = {0, 0, 0, 0};
+  if (dict.count("pad")) {
+    int pad = loadInt(dict["pad"]);
+    std::vector<size_t> pads;
+    for (size_t i = 0; i < 4; i++) {
+      pads.push_back(pad);
+    }
+    return pads;
+  } else if (dict.count("pad_t")) {
+    std::vector<size_t> pads;
+    pads.push_back(loadInt(dict["pad_t"]));
+    assert(dict.count("pad_l") && "missing pad_l");
+    pads.push_back(dict.count("pad_l") ? loadInt(dict["pad_l"]) : 0);
+    assert(dict.count("pad_b") && "missing pad_b");
+    pads.push_back(dict.count("pad_b") ? loadInt(dict["pad_b"]) : 0);
+    assert(dict.count("pad_r") && "missing pad_r");
+    pads.push_back(dict.count("pad_r") ? loadInt(dict["pad_r"]) : 0);
+  }
+  return zeroPads;
```

Comment:
It's better to move variable closer to where it is used. Or even better: `return {0, 0, 0, 0}`

Also, please add a comment, that default value for pad is 0, otherwise unprepared reader may be confused, that we return 0s if we don't see what we expect (and don't crash).